### ISSUE TITLE
npm install gives old version of mongoose: 4.13.20 instead of 5.8.5

### ISSUE THREAD

**Issue by sajalk95** [Automattic/mongoose#8488]
Installing mongoose using npm (v 6.13.4) downloads mongoose v 4.13.20. 
Has been happening since the last few hours only. 
This creates problems with the mongoose.connect also while using Mongo Atlas server.

**Comment by teolag** [comment]
Yeah....  something seems strange.

`npm outdated` wants to downgrade to 4.13.20
```
Package   Current  Wanted   Latest  Location
mongoose    5.8.5   5.8.5  4.13.20  my-project
```

**Comment by KevinBusse** [comment]
I guess @vkarpov15 accidentally published to latest instead of legacy:

```
λ npm view mongoose

mongoose@4.13.20 | MIT | deps: 13 | versions: 566
Mongoose MongoDB ODM
http://mongoosejs.com

keywords: mongodb, document, model, schema, database, odm, data, datastore, query, nosql, orm, db

dist
.tarball: https://registry.npmjs.org/mongoose/-/mongoose-4.13.20.tgz
.shasum: 7d6cdc35eca23306bb7388a74a08be1dc1487a7d
.integrity: sha512-lyOoXg/+S8VmW7gh816H9QcBSqLMCOXecVxflES729lb3qAU3W/AHLbCpHSIPFDJW5mXs4z5sCsjuLBa56w1Jg==
.unpackedSize: 1.0 MB

dependencies:
async: 2.6.0        lodash.get: 4.4.2   mquery: 2.3.3       sliced: 1.0.1
bson: ~1.0.4        mongodb: 2.2.34     ms: 2.0.0
hooks-fixed: 2.0.2  mpath: 0.5.1        muri: 1.3.0
kareem: 1.5.0       mpromise: 0.5.5     regexp-clone: 0.0.1

maintainers:
- aaron <aaron.heckmann+github@gmail.com>
- rauchg <rauchg@gmail.com>
- tjholowaychuk <tj@vision-media.ca>
- vkarpov15 <val@karpov.io>

dist-tags:
latest: 4.13.20  legacy: 4.13.19  unstable: 3.9.7

published 8 hours ago by vkarpov15 <val@karpov.io>
```

**Comment by Icesofty** [comment]
Temporary install solution :

`npm i mongoose@5.8.5`

**Comment by vkarpov15** [comment]
I'm so sorry for the trouble - I thought I published 5.8.6 yesterday to overwrite that :( Just published 5.8.6.

We haven't published 4.x in a while, so we were rusty.

**Comment by Nirjus** [comment]
good
